app/helpers/application_helper/button/cloud_subnet_new.rb

```diff
@@ -12,6 +12,6 @@ def role_allows_feature?
 
   # disable button if no active providers support create action
   def disabled?
-    ::EmsNetwork.all.none? { |ems| CloudSubnet.class_by_ems(ems).supports_create? }
+    ::EmsNetwork.all.none? { |ems| CloudSubnet.class_by_ems(ems).supports?(:create) }
```

Comment:
I feel like this is safer code, but not sure if we want to change anything at this time.

```suggestion
    ::EmsNetwork.all.none? { |ems| CloudSubnet.class_by_ems(ems)&.supports?(:create) }
```

Also feel like we need a pattern to simplify this:

not sure what you call `class_by_ems` + `supports`

```ruby
ems.class_supports?("CloudSubnet", :create)
```